README.md

```diff
@@ -0,0 +1,279 @@
+<a name="readme-top"></a>
+
```

Comment:
- [ ] This is the page I see when I visit your app. Your navigation does not work so I am unable to add any books. Kindly ensure your app works and that changes are persisted in the local storage.

  ![image](https://user-images.githubusercontent.com/81468675/218694217-cbd47288-3ff6-4966-851a-b443aa0a4bd4.png)